studio/components/layouts/TableEditorLayout/TableEditorMenu.tsx

```diff
@@ -105,18 +111,20 @@ const TableEditorMenu = ({
     >
       {/* Schema selection dropdown */}
       <div className="px-3 mx-4">
-        {!meta.schemas.isInitialized ? (
+        {isSchemasLoading && (
```

Comment:
btw lets do the same thing here for the database/tables page - as a workaround for the odd listbox problem

Reply:
its better too since it'll be an actual loading state

Reply:
Yep will do this in follow-up PRs :)